src/core/generator/clones/clonesGenerator.ts

```diff
@@ -179,6 +200,16 @@ const createIconClone = async (
                   : clonesConfig.light!.folderNamesExpanded!;
           folderNamesCfg[folderName] = clone.name;
         });
+      } else if (isLanguage(cloneOpts)) {
+        // sets the associated language ids for the cloned icon
+        cloneOpts.ids?.forEach((langId) => {
+          const languageIdCfg =
+            clone.variant === Variant.Base
+              ? clonesConfig.languageIds!
+              : clonesConfig.light!.languageIds!;
+
+          languageIdCfg[langId] = clone.name;
```

Comment:
This will throw an error when trying to set the light variant of the icon in line 211 because `clonesConfig.light.languageIds` will be `undefined`

To fix this, we need to add `languageIds` to `src\core\generator\clones\utils\cloning.ts` in the `createCloneConfig` function.


```diff
export const createCloneConfig = () => {
  const manifest = createEmptyManifest();
  manifest.light = {
    fileExtensions: {},
    fileNames: {},
    folderNames: {},
    folderNamesExpanded: {},
+   languageIds: {},
  };

  return manifest;
};
```

Reply:
Other than that, everything else it's working and LGTM!

Reply:
nice catch and thank you for reviewing 😊 I've added the missing `languageIds`.